include/openssl/ssl.h

```diff
@@ -1899,6 +1899,15 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
 # ifndef OPENSSL_NO_CT
 
 /*
+ * A callback for verifying that the received SCTs are sufficient.
+ * Expected to return 1 if they are sufficient, otherwise 0.
+ * May return a negative integer if an error occurs.
```

Comment:
The negative return was part of Rob's original interface.  Since he never published documentation (not so subtle hint), I'm just cargo-culting behaviour.

As for "will be aborted" that's not the case.  With SSL_VERIFY_NONE the connection will not be aborted regardless of the callback's return value.  Same as with X.509 chain verification callbacks.